backend/backend_pre_start.py

```diff
@@ -0,0 +1,55 @@
+from tenacity import after_log, before_log, retry, stop_after_attempt, wait_fixed
+from sqlalchemy import text
+import logging
+from session import SessionLocal
```

Comment:
I'm getting this error:
```
server             | Traceback (most recent call last):
server             |   File "/app/backend/./backend_pre_start.py", line 4, in <module>
server             |     from session import SessionLocal
server             | ImportError: cannot import name 'SessionLocal' from 'session' (/app/backend/session.py)
```
Doesn't look like there's a method in `session.py`

Reply:
Yes, punting this for now.